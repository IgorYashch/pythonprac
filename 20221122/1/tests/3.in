!slkgdjldsfjgfdsjg;lkfdjslg;kdsjf;lkgfdj;lgjds;lkfgj;lfdsgj;fdsljglkfdsjg;lkfdjlkg;djfs;lkgjfds;lkgj;lfdsjg;lkfdjsdpfuqrioehfdbvshguriefdslanshiuerao;fdslkz.vnksjgiurajkfdsnsbhlgraolkfdsvnjlgiuraofjkdsjsgriuaofdsjkzvnjlgshiurafjdslkznjlshgiurioafsdnvblrisfnjmecw;oslkdvfnsbgihfadjcmv mnbaisdfkjvangjfesfdvjra;fsja;ijgerfdsnvajfgjnksjfdgras;fjvzbrjksfd;kfdg