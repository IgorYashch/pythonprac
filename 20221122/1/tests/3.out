!;lkgj;lfdsjg;lkfdjsdpfuqrioehfdbvshguribhlgraolkfdsvnjlgiuraofjkdsjsgriuaofdsdd;efdslanshiuerao;fdslkz.vnksjgiurajkfdsnsesfdvjra;fsja;ijgefadjcmv mnbaisdfkjvangjffdsljglkfdsjg;lkfdjlkg;djfs;lkgjfdsfjvzbrgjf;lkgfdj;lgjds;lkfgj;lfdsgj;jgfdsjg;lkfdjslg;kdsjksfjkzvnjlgshiurafjdslkznjlshgiurioafkfrfdsnvajfgjnksdnvblrisfnjmecw;oslkdvfnsbgihsjfdgras;slkgdjldsf
